dev/com.ibm.ws.logging_2_fat/publish/servers/com.ibm.ws.logging.ffdcCleanup/server.xml

```diff
@@ -15,7 +15,7 @@
     <include location="../fatTestPorts.xml"/>
     
     <featureManager>
-	   <feature>jsp-2.2</feature>
```

Comment:
Do we also need to add `timedexit-1.0` here?

Reply:
The ffdcCleanup server is not being used in this PR's changes, I think we can remove this file as well.